### PR TITLE
Improved documentation style

### PR DESCRIPTION
**Pull request by ahcorde**
Improved documentation style

@brawner

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

**Comment by scpeters** [comment]
I just merged #73; can you rebase this?

**Comment by ahcorde** [comment]
@scpeters rebased!

File: include/console_bridge/console.h [hunk 1]
```diff
@@ -91,37 +91,37 @@ enum CONSOLE_BRIDGE_DLLAPI LogLevel
...
-  }
-
-  /** \brief log a message to the output handler with the given text
-      and logging level from a specific file and line number */
+  OutputHandler(void){}
+
+  virtual ~OutputHandler(void){}
+
+  /**
+   * \brief log a message to the output handler with the given text       and logging level from a
```
Comment (scpeters) [inline include/console_bridge/console.h:108]: some extra spaces on this line

File: include/console_bridge/console.h [hunk 2]
```diff
@@ -142,41 +142,51 @@ class CONSOLE_BRIDGE_DLLAPI OutputHandlerFile : public OutputHandler
   FILE *file_;
 };
 
-/** \brief This function instructs ompl that no messages should be outputted.
+/**
+ * \brief This function instructs ompl that no messages should be outputted.
```
Comment (scpeters) [inline include/console_bridge/console.h:146]: I think this shouldn't say `ompl`

**Pull request merged**